manywheel/build_docker.sh

```diff
@@ -54,11 +53,11 @@ case ${GPU_ARCH_TYPE} in
         ;;
     cuda-aarch64)
         TARGET=cuda_final
-        DOCKER_TAG=cuda-aarch64
+        DOCKER_TAG=cuda${GPU_ARCH_VERSION}
```

Comment:
Would we need "_" in between as the tag is "cuda-aarch64"? i.e: DOCKER_TAG=cuda_${GPU_ARCH_VERSION}